src/frontend/efiling-frontend/.env.production

```diff
@@ -0,0 +1,2 @@
+SKIP_PREFLIGHT_CHECK=true
+DISABLE_ESLINT_PLUGIN=true
```

Comment:
Are these two environment variables needed to bypass an issue?

Reply:
Yes. This is the only successful way to get around the airbnb issue. It would be resolved by updating to react-scripts 5.0.0 but as it stands, doing so creates a webpack error that does not have a fix yet.